dotCMS/src/integration-test/java/com/dotcms/publishing/PushPublishFiltersInitializerTest.java

```diff
@@ -0,0 +1,58 @@
+package com.dotcms.publishing;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotcms.util.YamlUtil;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.util.Logger;
+import com.google.common.collect.ImmutableMap;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Map;
+import java.util.stream.Stream;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class PushPublishFiltersInitializerTest {
+
+    private static PushPublishFiltersInitializer pushPublishFiltersInitializer;
+    private static File path;
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5458951)